store/schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE signature_requests (
+    incoming_type character varying(40) NOT NULL,
```

Comment:
what are examples of incoming_type values? It seems like having the second column called incoming_ethereum_transaction_hash implies this will only be used for ethereum

Reply:
The table fields are designed to allow other chains in the future. If we add Bitcoin, `incoming_type` will be equal `bitcoin` and there will be a new nullable field: `incoming_bitcoin_transaction_hash`.

Reply:
@bartekn if incoming_type specifies the chain then it seems like `incoming_ethereum_transaction_hash` should jut be renamed to `transaction_hash` since it's not possible to have a row with both `incoming_ethereum_transaction_hash` and `incoming_bitcoin_transaction_hash`

Reply:
I think we should have a separate field for each network. For example Solana use hashes that are longer than 64 chars.

Reply:
@tamirms changed to use a single column. The hash column was text anyway and length checks aren't that important because we load/check the hash anyway.